CorsixTH/Lua/epidemic.lua

```diff
@@ -599,11 +599,11 @@ end
 function Epidemic:getBestVaccinationTile(nurse, patient)
   -- General usage tile finder, used in the other cases
   -- when the patient isn't sitting on a bench
-  local function get_best_usage_no_bench(nurse,patient)
+  local function get_best_usage_no_bench(nurse2, patient2)
```

Comment:
Possibly promote the function instead. It does use the closure for self, so the entity map would have to be passed in another way.